README.md

```diff
@@ -1 +1,10 @@
 # AlarmClock
+
+Install wiht:
+
+~~~Smalltalk
+Metacello new
+    baseline: 'AlarmClock';
+    repository: 'github://tinchodias/pharo-test-repo';
```

Comment:
BTW, I should rename the project.

Reply:
Good idea, let's create a new issue to remember.